src/ol/source/tile.js

```diff
@@ -204,6 +204,19 @@ ol.source.Tile.prototype.getTile = function(z, x, y, pixelRatio, projection) {};
 
 
 /**
+ * Get the interim tile most suitable for rendering.
+ * @param {!ol.Tile} tile Most recent tile (head of the interim tile list)
+ * @return {!ol.Tile} Best tile for rendering at this location.
+ *
+ */
+ol.source.Tile.prototype.getInterimTile = function(tile) {
```

Comment:
This should not be necessary. Instead, the tiles consumed by `ol.source.TileImage` should be typed `ol.ImageTile` instead of `ol.Tile`.